### PR TITLE
Update daemonset.yaml

### PR DESCRIPTION
**Pull request by master-musi**
Enable opportunity to add other environment variables

### CODE REVIEW

**Comment by paologallinaharbur** [comment]
Hello, you missed updating the readme https://github.com/newrelic/nri-kubernetes/tree/v2/charts/newrelic-infrastructure

File: charts/newrelic-infrastructure/values.yaml [hunk 1]
```diff
@@ -185,6 +185,11 @@ openshift:
 # Ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/
 # priorityClassName: high-priority
 
+# Allows to add additional environment variables
+# extraEnv:
```
Comment (paologallinaharbur) [inline charts/newrelic-infrastructure/values.yaml:189]: could you follow the same approach in the values https://github.com/newrelic/nri-kubernetes/blob/e1cbd0007575f47296afe88534fc924c5e9bdd60/charts/newrelic-infrastructure/values.yaml#L99 ?

Comment (master-musi) [reply charts/newrelic-infrastructure/values.yaml:189]: Hello! Updated PR, please review

Comment (master-musi) [reply charts/newrelic-infrastructure/values.yaml]: Updated the PR, please review

**Comment by master-musi** [comment]
Hello!
I have updated the PR, please review 
--
Sent from Mail.ru Monday, 26 September 2022, 01:34AM -07:00 from PaoloGallina  ***@***.*** :

**Review by paologallinaharbur — APPROVED** [review]

**Pull request merged**